### ISSUE TITLE
Collect data from multiple Bluetooth scales at once;

### ISSUE THREAD

**Issue by krisjans** [oliexdev/openScale#770]
**Is your feature request related to a problem? Please describe.**
I own two scales (Beurer BF600 and Beurer BF850) and testing latest changes on both of them is a bit chore - constantly go to settings and change selected scale.
For someone who is living in several places (for example in city during week and outside city during week-end) this might even  be realistic scenario to use two or more scales.
Also currently the consent code is randomized and not equal on both scales - therefore you have to enter pin whenever you change scale. And if user index on scale does not match then this will simply wont  work.

**Describe the solution you'd like**
In openScale settings allow to select multiple Bluetooth devices to use.
When user press `Bluetooth connect` button, then connect to the scale that is in range. If both of them are in range, then connect to both of them and collect latest measurements.

**Comment by oliexdev** [comment]
I agree that would be a good enhancement but it would a lot of work to implement that.

**Comment by oliexdev** [comment]
Since there has been no progress on this issue for a long time, I am closing it for now.
